Docs/Userdocs/sm5_migration.md

```diff
@@ -0,0 +1,37 @@
+StepMania 5.1 to ITGmania Migration Guide
+=========================================
+
+This guide explains how to migrate from StepMania 5.1 to ITGmania without losing access to your settings and songs.
+
+The steps are as follows:
+
+1. Install ITGmania.
+2. Copy over songs, settings, profiles and other user data. (See detailed instructions below.)
+   If you don't intend on keeping StepMania 5.1 the data can be moved instead of copying.
+3. (skip for portable builds) ITGmania ships with Simply Love and the Club Fantastic song packs included. Duplicates of them can be removed from the user data directory.
+4. Uninstall StepMania 5.1 (optional).
+   If you choose to do so you can also uninstall the GrooveStats launcher as ITGmania has network functionality built in and doesn't need an external tool for that.
+5. Make sure to enable GrooveStats integration in the operator menu if you want to use it.
+6. You are all set!
+
+If you want to keep StepMania 5.1 alongside ITGmania you can save some space by using the `AdditionalSongFolders` setting to point both installations to the same songs folder.
+
+## Portable builds (all platforms)
```

Comment:
I would add a line that says "Create a file after installation called `Portable.ini` at the root directory of the install to create a portable build." (or something similar)

Reply:
added!